report/src/presence_technologies/bluetooth.tex

```diff
@@ -0,0 +1,44 @@
+\section{Bluetooth} \label{sec:bluetooth}
+
+Another possible technology for presence detection is Bluetooth.
+Bluetooth is a common technology used in many devices such as smartphones, tablets, and laptops.
+Because it is a standardized technology, all devices that support Bluetooth can communicate with each other.
+
+There are two different types of radio signals used in Bluetooth: Classic and Low Energy (LE).
+They differ in terms of power consumption, range, and data transfer rate.\cite{BluetoothSIG}
+
+\begin{figure}[h!]
+  \centering
+  \includegraphics[width=1\textwidth]{Bluetooth_Technology_Overview_Graphic.png}
+  \caption{Overview of the Bluetooth Classic and Low Energy technologies\cite{BluetoothOverview}.}
+  \label{fig:classic_vs_LE}
+\end{figure}
+
+\subsection{Bluetooth Classic} \label{sec:bluetooth_classic}
+The Bluetooth Classic radio is the original version of the Bluetooth technology introduced in 1999 for point-to-point device communication.
+It is used for data transfer between devices but is primarily used for wireless audio streaming across devices because of its high data transfer rate.
+According to the Bluetooth Special Interest Group (SIG), the data transfer rate of Bluetooth Classic is 3 Mb/s.
+
+Wireless devices such as headphones, speakers, and car stereos often use Bluetooth Classic to stream audio to and from a smartphone.
+However, the relatively high power consumption of Bluetooth Classic makes it unsuitable for presence detection using battery-powered devices.
+Furthermore, it is quite sensitive to obstructions, making it less suitable for presence detection compared to Bluetooth Low Energy.\cite{BluetoothSIG}
+
+\subsection{Bluetooth Low Energy} \label{sec:bluetooth_low_energy}
+Bluetooth Low Energy (BLE) is a newer version of the Bluetooth technology introduced in 2010 designed for low power consumption.
+In contrast to Bluetooth Classic, the data transfer rate of BLE is much lower.
+This makes it more usable in battery-powered devices such as smartphones, smartwatches, and fitness trackers.
```

Comment:
sources for all of these paragraphs. Cite the source at the end of each paragraph if the source is for that paragraph. Keep the paragraphs but cite same source multiple times rather than one big text.

Reply:
This goes for everything below.